frontend/src/view/UserComment/CommentContainer.styled.js

```diff
@@ -0,0 +1,3 @@
+import styled from 'styled-components';
+
+export const CommentContainer = styled.div``;
```

Comment:
Jeśli są to style jednego pliku komponentu to powinny być w jednym pliku styled.js, odpowiadającego nazwie plikowi .js
Tutaj, `UserComment/UserComment.styled.js `

Reply:
Okej, poprawię